lesson19.js

```diff
@@ -0,0 +1,129 @@
+//Return total days count
+
+//getDaysInYear(year);
+const getDaysInYear = (year) => {
+  return year % 4 === 0 ? 366 : 365;
```

Comment:
it's not a full formula

1. If the year is evenly divisible by 4, go to step 2. Otherwise, go to step 5.
2. If the year is evenly divisible by 100, go to step 3. Otherwise, go to step 4.
3. If the year is evenly divisible by 400, go to step 4. Otherwise, go to step 5.
4. The year is a leap year (it has 366 days).
5. The year is not a leap year (it has 365 days).